### ISSUE TITLE
Cannot update Debian stable release as its signed by an expired GPG key

### ISSUE THREAD

**Issue by michaelkmccormick28** [apache/incubator-pagespeed-mod#1872]
Trying to update packages on Ubuntu 18.04.2 LTS (GNU/Linux 4.15.0-47-generic x86_64) results in this error for the PageSpeed Apache Module.

```
Get:11 http://dl.google.com/linux/mod-pagespeed/deb stable Release [2,156 B]
Get:12 http://dl.google.com/linux/mod-pagespeed/deb stable Release.gpg [819 B]
Ign:12 http://dl.google.com/linux/mod-pagespeed/deb stable Release.gpg
Reading package lists... Done
W: GPG error: http://dl.google.com/linux/mod-pagespeed/deb stable Release: The following signatures were invalid: EXPKEYSIG 1397BC53640DB551 Google Inc. (Linux Packages Signing Authority) <linux-packages-keymaster@google.com>
E: The repository 'http://dl.google.com/linux/mod-pagespeed/deb stable Release' is not signed.
```

**Comment by oschaaf** [comment]
I think it's worth trying again after a little while, this probably will get fixed automagically.
There's not much we can do about it here, closing this.

**Comment by zaphod-42** [comment]
I had this issue today as well and corrected it with the steps outlined here:

https://support.plesk.com/hc/en-us/articles/360021566754--apt-update-is-not-working-on-Plesk-server-with-Ubuntu-or-Debian-after-installing-PageSpeed-Apache-module-through-Google-PageSpeed-Insights-extension-The-repository-http-dl-google-com-linux-mod-pagespeed-deb-stable-Release-is-not-signed

**Comment by michaelkmccormick28** [comment]
This has now been re-signed with a new key so yes you can update it.

**Comment by tbergeron** [comment]
Same happens today:

```
Ign:8 http://dl.google.com/linux/mod-pagespeed/deb stable Release.gpg
Reading package lists... Done
W: GPG error: http://dl.google.com/linux/mod-pagespeed/deb stable Release: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 6494C6D6997C215E
E: The repository 'http://dl.google.com/linux/mod-pagespeed/deb stable Release' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```
I guess it'll get fixed automagically again? :yum:
